embassy-nrf/src/buffered_uarte.rs

```diff
@@ -182,6 +183,61 @@ impl<'d, U: UarteInstance, T: TimerInstance> BufferedUarte<'d, U, T> {
         }
     }
 
+    #[cfg(not(any(feature = "_nrf9160", feature = "nrf5340")))]
+    fn apply_workaround_for_enable_anomaly() {
+        // Do nothing
+    }
+
+    #[cfg(any(feature = "_nrf9160", feature = "nrf5340"))]
+    fn apply_workaround_for_enable_anomaly() {
```

Comment:
Could you keep this fn in one place instead of duplicating it? Maybe a standalone fn in the `uarte` mod that takes the regblock as an argument, so it can be used by both.
Thanks!

Reply:
ah yes - wilco